src/compiler/gcc.rs

```diff
@@ -402,6 +402,9 @@ pub fn generate_compile_commands(path_transformer: &mut dist::PathTransformer,
                                 env_vars: &[(OsString, OsString)])
                                 -> Result<(CompileCommand, Option<dist::CompileCommand>, Cacheable)>
 {
+    #[cfg(not(feature = "dist-client"))]
+    let _ = path_transformer;
```

Comment:
I don't love statement-level `#[cfg]` usage like this, but I don't know that there's a better pattern. Maybe using [`cfg-if`](https://github.com/alexcrichton/cfg-if) would make it nicer?